pages/_app.tsx

```diff
@@ -1,84 +1,57 @@
-import React, { useCallback, useEffect, useState } from 'react'
-import App from 'next/app'
-import type { AppProps, AppContext } from 'next/app'
-import 'react-toastify/dist/ReactToastify.css'
-import '@/styles/globals.css'
-import { useRouter } from 'next/router'
-import { ToastContainer } from 'react-toastify'
-import Head from 'next/head'
-import styled from 'styled-components'
-import Link from 'next/link'
-import { Layout, Space } from 'antd'
-import sv from '@/constants/styles'
-import { isNil } from 'ramda'
-import Loading from '@/components/elements/Loading'
-import { WalletProvider } from '@/modules/wallet'
-import { StorefrontProvider } from '@/modules/storefront'
-import SocialLinks from '@/components/elements/SocialLinks'
-import useWindowDimensions from '@/hooks/useWindowDimensions'
-
-const GOOGLE_ANALYTICS_ID = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS_ID
-
-const { Header, Content } = Layout
-
-const HeaderTitle = styled.div`
-  font-size: 24px;
-  line-height: 2px;
-  font-weight: 900;
-  margin-right: auto;
-  a {
-    color: ${sv.colors.buttonText};
-    &:hover {
-      color: ${sv.colors.buttonText}
-    }
-  }
-`
-
-const AppHeader = styled(Header)`
-  ${sv.flexRow};
-  margin: 0 0 40px 0;
-`
-
-const HeaderLinkWrapper = styled.div<{ active: boolean; }>`
-  color: ${sv.colors.buttonText};
-  ${({ active }) => active && `text-decoration: underline;`}
-`
+import React, { useEffect } from 'react';
+import type { AppProps } from 'next/app';
+import 'react-toastify/dist/ReactToastify.css';
+import '@/styles/globals.css';
+import { useRouter } from 'next/router';
+import { ToastContainer } from 'react-toastify';
+import Head from 'next/head';
+import styled from 'styled-components';
+import { Layout } from 'antd';
+import { isNil } from 'ramda';
+import Loading from '@/components/elements/Loading';
+import { WalletProvider } from '@/modules/wallet';
+import { StorefrontProvider } from '@/modules/storefront';
+import { Content } from 'antd/lib/layout/layout';
+import { AppHeader } from '@/components/elements/AppHeader';
+import { route } from 'next/dist/next-server/server/router';
+
+const GOOGLE_ANALYTICS_ID = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS_ID;
 
 const AppLayout = styled(Layout)`
   overflow-y: auto;
-`
+`;
 
 function MyApp({ Component, pageProps }: AppProps) {
-  const router = useRouter()
-  const windowDimensions = useWindowDimensions();
+  const router = useRouter();
+  const showHeader = router.pathname !== '/bulk-upload/0' && router.pathname !== '/bulk-upload/1';
```

Comment:
We should still report page views on these changes. It will help understand how deep people go into the process.

Reply:
I'll add this to my Analytics ticket https://holaplex.atlassian.net/jira/software/projects/HOLA/boards/2?selectedIssue=HOLA-74